contracts/transfer/circuits/src/gadgets/balance.rs

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) DUSK NETWORK. All rights reserved.
+// Licensed under the MPL 2.0 license. See LICENSE file in the project root for details. 
```

Comment:
Put the correct headers. See: https://github.com/dusk-network/dusk-pki/blob/0aeaf14ba49db6a96fba6532f6f174cdd8d1d9b4/src/decode.rs#L1